packages/docusaurus/src/commands/start.ts

```diff
@@ -236,7 +236,20 @@ export async function start(
   );
 
   const devServer = new WebpackDevServer(devServerConfig, compiler);
-  devServer.startCallback(() => {
+  devServer.startCallback(async () => {
+    // Plugin Lifecycle - postBuild.
```

Comment:
```suggestion
    // Plugin Lifecycle - postStart.
```

Reply:
Thanks for catching this. Dyslexia really made me think I changed this comment 🤦